.github/workflows/release_to_draft.yml

```diff
@@ -0,0 +1,164 @@
+name: "Release matrix"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Specify tag to create'
+        required: true
+
+jobs:
+  
+  release:
+    name: "Release"
+    strategy:
+      matrix:
+        os: [windows, ubuntu, macos] 
+        gfx: [vulkan, opengl]
```

Comment:
change `vulkan` to `wgpu` since vulkan is technically only used on linux, and metal for macos / directx12 for windows

Reply:
Will rename!